docs/pages/templates/marketplace/index.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Starter Kit'
+---
+
+import { Cards } from 'nextra-theme-docs'
+import { Card } from '../../../components/Card'
+
+# Starter kit
+
+The starter kit is fully fledged NFT marketplace running on the Liteflow infrastructure to showcase the uses of the infrastructure and/or be used as a starter kit to launch a product fast.
+
+This application is fork-able on this repository: https://github.com/liteflow-labs/starter-kit.
+
+<Cards num={2}>
+  <Card noPadding>![Home](/templates/marketplace/home.png)</Card>
+  <Card noPadding>![Home](/templates/marketplace/explore.png)</Card>
+  <Card noPadding>![Home](/templates/marketplace/detail.png)</Card>
+  <Card noPadding>![Home](/templates/marketplace/profile.png)</Card>
+</Cards>
+
```

Comment:
### Open Source
This application is open-source and fork-able. You can view the source code and contribute to it on: https://github.com/liteflow-labs/starter-kit.